.travis.yml

```diff
@@ -1,57 +1,57 @@
 language: python
 matrix:
   include:
-    - stage: client unit tests
-      python: 3.4
-      env:
-        - TEST_SUITE="tests/uts/test_client.py"
-    - python: 3.5
-      env:
-        - TEST_SUITE="tests/uts/test_client.py"
-    - python: 3.6
-      env:
-        - TEST_SUITE="tests/uts/test_client.py"
-    - python: 3.7
-      env:
-        - TEST_SUITE="tests/uts/test_client.py"
-    - stage: agent unit tests
-      python: 2.7
-      env:
-        - TEST_SUITE="tests/uts/test_agent.py"
-    - stage: integration tests
-      python: 3.4
-      env:
-        - TEST_SUITE="tests/integration"
-      before_install:
-        - tests/integration/setup.sh
-    - python: 3.5
-      env:
-        - TEST_SUITE="tests/integration"
-      before_install:
-        - tests/integration/setup.sh
-    - python: 3.6
-      env:
-        - TEST_SUITE="tests/integration"
-      before_install:
-        - tests/integration/setup.sh
-    - python: 3.7
-      env:
-        - TEST_SUITE="tests/integration"
-      before_install:
-        - tests/integration/setup.sh
+  - stage: client unit tests
+    python: 3.4
+    env:
+    - TEST_SUITE="tests/uts/test_client.py"
+  - python: 3.5
+    env:
+    - TEST_SUITE="tests/uts/test_client.py"
+  - python: 3.6
+    env:
+    - TEST_SUITE="tests/uts/test_client.py"
+  - python: 3.7
+    env:
+    - TEST_SUITE="tests/uts/test_client.py"
+  - stage: agent unit tests
+    python: 2.7
+    env:
+    - TEST_SUITE="tests/uts/test_agent.py"
+  - stage: integration tests
+    python: 3.4
+    env:
+    - TEST_SUITE="tests/integration"
+    before_install:
+    - tests/integration/setup.sh
+  - python: 3.5
+    env:
+    - TEST_SUITE="tests/integration"
+    before_install:
+    - tests/integration/setup.sh
+  - python: 3.6
+    env:
+    - TEST_SUITE="tests/integration"
+    before_install:
+    - tests/integration/setup.sh
+  - python: 3.7
+    env:
+    - TEST_SUITE="tests/integration"
+    before_install:
+    - tests/integration/setup.sh
 script: pytest $TEST_SUITE -v
 dist: xenial
 install:
-  - pip install .[test]
+- pip install .[test]
 deploy:
   provider: pypi
-  user: wekaio-pypi
+  user: __token__
```

Comment:
Where is this coming from?

Reply:
commit hash: `fc20ee4`
https://docs.travis-ci.com/user/deployment/pypi/